### PR TITLE
Proposal of new merge distro and sw

### PR DESCRIPTION
**Pull request by cpriouzeau**
This pull request propose a new source tree of meta-ledge-distro.

Meta-ledge-distro have:
- a distro ledge-distro inherited from rpb with minor addons
- all sw stuff
- specific packages group: network and optee
- two images inherited from rpb.

rpb: meta-rpb from linaro.

And realignment of images content with content of lmp-gateway-image content.

### CODE REVIEW

File: meta-ledge-distro/recipes-connectivity/openavnu/openavnu-daemons/run_openavnu_daemon.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+
+nic=$1
+echo "Starting daemons on "$nic
+
+groupadd ptp > /dev/null 2>&1
+
+# start gptp
+/usr/bin/gptp $nic &
```
Comment (muvarov) [inline meta-ledge-distro/recipes-connectivity/openavnu/openavnu-daemons/run_openavnu_daemon.sh]: better to use PATH the absolute path to binary

Comment (cpriouzeau) [reply meta-ledge-distro/recipes-connectivity/openavnu/openavnu-daemons/run_openavnu_daemon.sh]: This script come from the documentation of openavnu and with openembedded, the implicit rule are to call  the binary with an absolute except for the system tools (useradd, groupadd, ls, ...)

Comment (apalos) [reply meta-ledge-distro/recipes-connectivity/openavnu/openavnu-daemons/run_openavnu_daemon.sh]: The purpose of the PR is to revamp some dir structure. We can rewrite/adjust scripts later on based on member requirements. @muvarov has a point though here. If a member includes the recipe and changes the installation dir, he will have to change the scripts as well

File: meta-ledge-distro/recipes-samples/packagegroups/packagegroup-ledge-optee.bb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+inherit packagegroup
+
+RDEPENDS_packagegroup-ledge-optee = "\
+    optee-test \
+    optee-client\
+    \
+    optee-ledge \
+    "
+
```
Comment (apalos) [inline meta-ledge-distro/recipes-samples/packagegroups/packagegroup-ledge-optee.bb]: on the meta-ledge-bsp, in machines we got something like 
MACHINE_EXTRA_RRECOMMENDS_append_ledge-ti-am572x = " optee-os optee-client optee-test"
Can this be changed to MACHINE_EXTRA_RRECOMMENDS only? This is the machine file for am572x do we need to add it as an option as well?

Comment (apalos) [reply meta-ledge-distro/recipes-samples/packagegroups/packagegroup-ledge-optee.bb]: MACHINE_FEATURES += "optee" is missing from machine definition for the TI board

**Comment by koenkooi** [comment]
For the distro config, either use plain rbp, or copy-and-rename the config files, a halfway solution makes everyone unhappy. After this change the ledge distro is just this:

```
DISTRO_NAME = "Reference-Platform-Build-for-LEDGE"
DISTRO_FEATURES_append = " virtualization "
```

The second entry doesn't do what you think it does, so it's just a name change. Ilias says that's just "for now" and bigger changes will come.

Try to keep external dependencies to a minimum, especially for core parts like your DISTRO.conf and reference images. 

and in general, please resist tweaking knobs just because they are there, like the IMAGE_LINGUAS stuff. Delete all that till there's an actual, demonstrable need for changes, not just copy/paste from files created years ago.

**Comment by cpriouzeau** [comment]
Pull request abandoned and replaced by another solution

**Pull request closed without merging**